conf/rest/9.12.0/qtree.yaml

```diff
@@ -11,8 +11,6 @@ counters:
   - ^export_policy.name   => export_policy
   - ^security_style       => security_style
   - id                    => id
-  - filter:
-      - name=!""
```

Comment:
As 22.05 behaviour would need all qtrees, I have removed this filter and later filtered with `exclude_equals`

Reply:
Yes, can be possible.

As the REST don't support merge of custom template, We can ask them to create this custom rest template as mentioned here https://github.com/NetApp/harvest/issues/1784, I will update the PR and the steps accordingly so no changes would be required in current template.

Reply:
Updated the PR, we don't need to change the steps mentioned in #1784.